frontend/messages/fr.json

```
@@ -1949,6 +1950,10 @@
 	"more": "Plus",
 	"activityCalendar": "Calendrier des activités",
 	"securityException": "Exception",
+	"securityAdvisories": "Avis de sécurité",
```

Biome (2.4.13)

[error] The key securityAdvisories was already declared.

(lint/suspicious/noDuplicateObjectKeys)